app/src/main/kotlin/edu/stanford/bdh/engagehf/observations/ObservationCollection.kt

```diff
@@ -6,4 +6,5 @@ enum class ObservationCollection(val collectionName: String, val loinc: Loinc) {
     HEART_RATE(collectionName = "heartRateObservations", loinc = Loinc.HEART_RATE),
     BODY_WEIGHT(collectionName = "bodyWeightObservations", loinc = Loinc.WEIGHT),
     BLOOD_PRESSURE(collectionName = "bloodPressureObservations", loinc = Loinc.BLOOD_PRESSURE),
+    SYMPTOMS(collectionName = "symptomScores", Loinc.N_A),
```

Comment:
feel free to update the loinc type to nullabe and in `ObservationCollection` and use `null` here. This is a separate use case and it is not ideal to introduce a `N/A` in the enum just because of this.